confluence_test.go

```diff
@@ -3,7 +3,7 @@ package confluence_test
 import (
 	"testing"
 
-	bfconfluence "github.com/kentaro-m/blackfriday-confluence"
+	bfconfluence "github.com/zabawaba99/blackfriday-confluence"
```

Comment:
I had to change this to get the tests working but accidentally committed it. Will remove.